spec_util/testdata/generalize_witnesses/spec.pb.txt

```diff
@@ -1474,7 +1474,7 @@ methods: <
     >
   >
   args: <
-    key: "jX1cqTk1FwI="
+    key: "Sajk40aPGIE="
```

Comment:
These are changing because they were previously incorrect, I'm assuming?  I know I've run into incorrect hashes in the unit test data before.

Reply:
Yes, they were incorrect. Pretty much why I added the sanity check to the test harness.